ivadomed/models.py

```diff
@@ -561,7 +592,7 @@ def forward(self, x):
         out = self.norm_lrelu_upscale_conv_norm_lrelu_l3(out)
 
         # Level 4 localization pathway
-        out = torch.cat([out, context_1], dim=1)
+        out = torch.cat([context_1, out], dim=1)
```

Comment:
Oh! Does it matter? Curious about this :-)

Reply:
Nope,  I saw both implementations and both work the same according to my tests :)